assets/webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
```

Comment:
We actually don't _need_ any JS in our Auth app. 
it should be 100% server-side rendered and no client-side validation (_for now..._).